Standards/CCSMB-5.md

```diff
@@ -0,0 +1,22 @@
+# CCSMB-5: Radio Transmissions
+```md
+*Author: Emma Beurskens <@emmaknijn>*
+*Version: v1.0.0*
+*Last updated: YYYY-MM-DD*
+```
+
+## Design
+A client will only listen to packets while the audio is being transmitted from a server
+The server will communicate to the client over modem
+
+## Packet
+A packet consists of the following fields in a table:
+- buffer: Signed 8-bit PCM @ 48kHz
+- id: A random number that identifies the radio station
```

Comment:
Is there an easy way to generate UUIDs in ComputerCraft? If there is, maybe we should use those.

Reply:
Unsure, let me do some research into that

Reply:
There is actually a UUID API but its really ancient: http://www.computercraft.info/forums2/index.php?/topic/13924-uuid-api-universally-unique-identifier/

Reply:
https://gist.github.com/jrus/3197011

This is all UUID4 requires in lua.

Reply:
but at the end of the day id is redundant since stationName exists